compute/cloud-client/src/test/java/compute/disks/HyperdisksIT.java

```diff
@@ -96,15 +96,15 @@ public void stage1_CreateHyperdiskTest()
     Assert.assertTrue(hyperdisk.getZone().contains(ZONE));
   }
 
-  @Disabled
   @Test
   public void stage1_CreateHyperdiskStoragePoolTest()
       throws IOException, ExecutionException, InterruptedException, TimeoutException {
     String poolType = String.format("projects/%s/zones/%s/storagePoolTypes/hyperdisk-balanced",
         PROJECT_ID, ZONE);
     StoragePool storagePool = CreateHyperdiskStoragePool
          .createHyperdiskStoragePool(PROJECT_ID, ZONE, STORAGE_POOL_NAME, poolType,
-         "advanced", 10240, 10000, 10240);
+         "advanced", 10240, 10000, 10240,
```

Comment:
please use `PERFORMANCE_PROVISIONING_TYPE` instead of `"advanced"` literal

Reply:
Here is variable capacityProvisioningType which also has value "advanced". I've fixed code  for new added variable PERFORMANCE_PROVISIONING_TYPE. Thank you!

Reply:
Please, replace `"advanced"` string literal with `PERFORMANCE_PROVISIONING_TYPE`:
```suggestion
         PERFORMANCE_PROVISIONING_TYPE, 10240, 10000, 10240,
```

Reply:
I made two variables to not mix them   
private static final String PERFORMANCE_PROVISIONING_TYPE = "advanced";
  private static final String CAPACITY_PROVISIONING_TYPE = "advanced";